### PR TITLE
[Port] Added Loading to Compute&Storage and Connection Strings Pages

### PR DESCRIPTION
**Pull request by nasc17**
Fixes https://github.com/microsoft/azuredatastudio/issues/13262

(Workaround for https://github.com/microsoft/azuredatastudio/issues/13134)

This adds loading indicators to the Compute&Storage and Connection Strings pages when the config is waiting for data to be present. 

loadingstring

Also...
TODO in Properties page was completed so that LinkKeyValue takes user to controller dashboard. (not currently shown on ADS)

### CODE REVIEW

File: extensions/arc/src/ui/dashboards/postgres/postgresComputeAndStoragePage.ts [hunk 1]
```diff
@@ -155,12 +161,13 @@ export class PostgresComputeAndStoragePage extends DashboardPage {
 						},
 						(_progress, _token) => {
 							return this._azdataApi.azdata.arc.postgres.server.edit(
-								this._postgresModel.info.name, this.saveArgs);
+								this._postgresModel.info.name, this.saveArgs).then(
```
Comment (bergeron) [inline extensions/arc/src/ui/dashboards/postgres/postgresComputeAndStoragePage.ts:164]: This change was moved into a separate PR right?

Comment (nasc17) [reply extensions/arc/src/ui/dashboards/postgres/postgresComputeAndStoragePage.ts:164]: Yup, https://github.com/microsoft/azuredatastudio/pull/13254 .

**Comment by Charles-Gagnon** [comment]
@nasc17 I moved this to be merged into the 1.24 release branch directly since we should have a fix in insiders here shortly so no need to merge this into main. 

(this also includes the other fixes you did for updating the link and waiting for edit refresh)

**Comment by Charles-Gagnon** [comment]
@chlafreniere This was approved in ship room - could you merge it in?
